src/action.c

```diff
@@ -286,7 +285,6 @@ actions_run(struct view *activator, struct server *server,
 			}
 			break;
 		case ACTION_TYPE_MOVE:
-			view = desktop_view_at_cursor(server);
```

Comment:
This causes a change in behavior when using `Move` from a keybind.
`Resize` from a keybind seems completely broken but that is a pre-existing issue (this PR would change the behavior of that as well though if it was working).

Maybe we could keep doing the `desktop_view_at_cursor()` calls if `view == NULL`?

We should likely also put a comment in there of the purpose of `desktop_view_at_cursor()` being important when using these actions from a keybind.

Reply:
Good catch, I'll fix this to match previous behavior for keybinds (`activator == NULL` case).